src/quantile.js

```diff
@@ -1,6 +1,19 @@
 import number from "./number.js";
+import identity from "./identity.js";
 
 export default function quantile(values, p, valueof = number) {
+  values = Array.from(values)
+    .map((d, i) => valueof(d, i, values))
+    .filter(v => !isNaN(v))
```

Comment:
This should use the `Float64Array.from(numbers(values, valueof))` technique from median.

Reply:
yes. moved numbers* to number.js to mutualize it

Reply:
You don’t need to filter since numbers will only give you not-NaN numbers.